test/src/main/java/io/strimzi/test/k8s/KubeClusterResource.java

```diff
@@ -358,4 +358,8 @@ public boolean fipsEnabled() {
         }
         return false;
     }
+
+    public boolean isMultiNode() {
```

Comment:
Isn't this already covered? Because we have annotation that checks that the cluster is multi node (or not).

Reply:
No it is not AFAIK. 
```java
int expectedNodeCount = annotation.get().workerNodeCount();

        KubeClusterResource clusterResource = KubeClusterResource.getInstance();

        if (clusterResource.client().getClusterWorkers().size() > expectedNodeCount) {    
 ```       
 this is inside such annotation